Week 4 Assignment Relaxr Blog/index.html

```diff
@@ -0,0 +1,72 @@
+			</header>
+			<div class="clearfix"></div>
+			<section class="alignleft">	
+				<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+				<img class="blog" src="./images/blog_photo1.jpg">
+				<p class="content"><span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur pretium id augue sit amet euismod. Vestibulum tempus rutrum mauris, quis varius augue molestie ac.</span>
+				<span class="content-hidden post-1"> Praesent id lacinia sapien. Vivamus efficitur mauris erat, sed gravida orci convallis ac. Praesent accumsan dolor massa. Proin vehicula feugiat lacinia. Nullam vitae sapien vel velit porttitor tempus id ac metus. Nullam varius nunc non cursus placerat. In eget est id eros porttitor efficitur. Mauris malesuada convallis est sed feugiat. Etiam pellentesque feugiat purus, sit amet iaculis nisi mollis sed. Nunc purus turpis, vestibulum nec dolor at, fringilla faucibus odio. Morbi felis massa, mattis eu neque sit amet, ultrices lobortis urna.</span></p>
+				<p class="more expand-1">Read more></p>
+				<p class="more content-hidden hide-1">Read less></p>
+				<hr/>
+				<h2>I traveled across the globe thanks to Relaxr’s Excel auomation</h2>
+				<img class="blog" src="./images/blog_photo2.jpg">
+				<p class="content"><span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur pretium id augue sit amet euismod. Vestibulum tempus rutrum mauris, quis varius augue molestie ac.</span><span class="content-hidden post-2"> Praesent id lacinia sapien. Vivamus efficitur mauris erat, sed gravida orci convallis ac. Praesent accumsan dolor massa. Proin vehicula feugiat lacinia. Nullam vitae sapien vel velit porttitor tempus id ac metus. Nullam varius nunc non cursus placerat. In eget est id eros porttitor efficitur. Mauris malesuada convallis est sed feugiat. Etiam pellentesque feugiat purus, sit amet iaculis nisi mollis sed. Nunc purus turpis, vestibulum nec dolor at, fringilla faucibus odio. Morbi felis massa, mattis eu neque sit amet, ultrices lobortis urna.</span></p>
+				<p class="more expand-2">Read more></p>
+				<p class="more content-hidden hide-2">Read less></p>
+			</section>
+			<aside class="alignright">
+				<h5>Categories</h5>
+				<ul>
+					<li><a href="#">Success stories</a></li>
+					<li><a href="#">Stats</a></li>
+					<li><a href="#">How tos</a></li>
+					<li><a href="#">Best of</a></li>
+					<li><a href="#">Business tips</a></li>
+				</ul>
+				<h5>About Relaxr</h5>
+				<p class="about"><span>Proin sed justo vel sapien varius auctor.</span>
+				<span class="content-hidden about-hidden"> Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</p></span>
+				<p class="more expand-3">Learn more</p>
+				<p class="more content-hidden hide-3">Less</p>
+				<a href="#"><img class="blog" src="./images/ad-unit.png"></a>
+			</aside>
+			<div class="clearfix"></div>
+			<footer class="yellowground">
+				<div class="button-navy">Sign Up Now!</div>
```

Comment:
This was originally an anchor tag, you changed it to a div.  It's better to keep buttons as ```<button>``` or ```<a>```.  In your case, ```event.preventDefault()``` doesn't do anything since clicking this button had no default behaviour.  Had it been a link, it would have prevented the browser from changing pages.